tools/generator/game.py

```diff
@@ -376,7 +380,8 @@ def explore(obj, schema, path):
     'constant': [
         {
             'cst_name': CONSTANT,
-            'cst_val': Union(int, float),
+            'cst_val': Union(int, str, float),
+            'cst_type': O(re.compile('(int|string|double)')),
```

Comment:
I don't like having a cst_type field because it's redundant with the type of the value in the yaml. I thought we removed that field, is it still used?

Reply:
AFAICT the last (and only) time it was used besides tests was [in 2016](https://github.com/prologin/prologin2016/blob/38fee9eac97818b1002d0579d3e4d9172c8dab3d/prologin2016.yml#L49) and there's only one constant that uses it. It could be necessary to implement #169, but I guess that since enum and struct field names must be unique, we could easily infer the type based on that.